packages/react-components/src/components/trace-context-component.tsx

```diff
@@ -197,8 +199,10 @@ export class TraceContextComponent extends React.Component<TraceContextProps, Tr
         this.props.messageManager.removeStatusMessage(this.INDEXING_STATUS_BAR_KEY);
         this.props.messageManager.removeStatusMessage(this.TIME_SELECTION_STATUS_BAR_KEY);
         this.props.removeResizeHandler(this.onResize);
+        signalManager().off(Signals.THEME_CHANGED, this.onBackgroundThemeUpdated);
```

Comment:
This doesn't seem related, unless you wanted to move the line to group these signal manager things together, in which case you forgot to delete it from line 198 above.